init_autoloader.php

```diff
@@ -4,20 +4,6 @@
     $loader = include 'vendor/autoload.php';
```

Comment:
In my opinion this is a good improvement! :) Would you push it in my feature branch?